doc/source/whatsnew/v0.21.1.txt

```diff
@@ -0,0 +1,83 @@
+.. _whatsnew_0211:
+
+v0.21.1
+-------
+
+This is a minor release from 0.21.1 and includes a number of deprecations, new
+features, enhancements, and performance improvements along with a large number
+of bug fixes. We recommend that all users upgrade to this version.
+
+.. _whatsnew_0211.enhancements:
+
+New features
+~~~~~~~~~~~~
+
+.. _whatsnew_0211.enhancements.other:
+
+Other Enhancements
+^^^^^^^^^^^^^^^^^^
+
+.. _whatsnew_0211.api_breaking:
+
+Backwards incompatible API changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0211.api:
+
+Other API Changes
+^^^^^^^^^^^^^^^^^
+
+.. _whatsnew_0211.deprecations:
+
+Deprecations
+~~~~~~~~~~~~
+
+.. _whatsnew_0211.prior_deprecations:
+
+Removal of prior version deprecations/changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0211.performance:
+
+Performance Improvements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0211.docs:
+
+Documentation Changes
+~~~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0211.bug_fixes:
+
+Bug Fixes
+~~~~~~~~~
```

Comment:
can you add some blank lines in each of the bug fix sub-sectios (and some of the other sections as well). this helps prevent conflicts (if people don't all pick the same line).